query/graphql/src/main/java/org/codice/ddf/admin/graphql/servlet/ExtendedOsgiGraphQLServlet.java

```diff
@@ -137,6 +138,11 @@ protected GraphQLErrorHandler getGraphQLErrorHandler() {
   }
 
   @Override
+  @SuppressWarnings({
+    "squid:S1135" /* Remove when TODO is completed */,
+    "squid:S1181" /* Catching throwable */,
+    "squid:S1989" /* Throwing exception in servlet */
```

Comment:
This last suppression is to have the methods throw exceptions instead of using try/catch. It was a low issue but throwing exceptions is not recommended in servlets. I suppressed it for now but I'm not sure if that's the way we want to go.